src/components/Card.tsx

```diff
@@ -64,9 +65,16 @@ const appStyles = classnames(
   display('flex'),
   flexDirection('flex-col'),
   alignItems('items-stretch'),
-  minHeight('sm:min-h-card', 'min-h-fit')
+  minHeight('sm:min-h-card', 'min-h-fit'),
+  maxHeight('max-h-app-card'),
```

Comment:
Not sure if we need all 3 `height, minHeight, maxHeight`, I think we can get rid of `height`

Reply:
Seems like mobile card is broken 
![image](https://user-images.githubusercontent.com/49658988/186415212-79e397f1-f585-4324-979a-006c11369497.png)

Reply:
Okay, this one seems to be fixed